test/widgets/content_test.dart

```diff
@@ -1047,8 +1047,7 @@ void main() {
     });
 
     testWidgets('fallback to displaying KaTeX source if unsupported KaTeX HTML', (tester) async {
-      await prepareContent(tester, plainContent(ContentExample.mathInlineUnknown.html));
-      tester.widget(find.text(r'\lambda'));
+      testContentSmoke(ContentExample.mathInlineUnknown);
```

Comment:
This is a test nested in a test, right? So the outer should become a group instead. (Like it does in the next commit anyway.)